build.gradle

```diff
@@ -166,6 +166,7 @@ dependencies {
     compile group: 'net.sourceforge.argparse4j', name: 'argparse4j', version: '0.7.0'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-base', version:'1.0.55'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-Formula', version:'1.0.64'
+    compile group: 'org.jetbrains', name: 'annotations', version:'15.0'
```

Comment:
Is it possible to use JSR 305 annotations instead? Netbeans (the IDE I use) doesn't support the jetbrains annotations as far as I know.